docs/resources/csms_event.md

```diff
@@ -0,0 +1,70 @@
+---
+subcategory: "Data Encryption Workshop (DEW)"
+---
+
+# huaweicloud_csms_event
+
+Manages a CSMS (Cloud Secret Management Service) event resource within HuaweiCloud.
+
+## Example Usage
+
+```hcl
+variable "notification_target_type" {}
+variable "notification_target_id" {}
+variable "notification_target_name" {}
+
+resource "huaweicloud_csms_event" "test" {
+  name                     = "test_name"
+  event_types              = ["SECRET_VERSION_CREATED", "SECRET_ROTATED"]
+  status                   = "ENABLED"
+  notification_target_type = var.notification_target_type
+  notification_target_id   = var.notification_target_id
+  notification_target_name = var.notification_target_name
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String, ForceNew) Specifies the region in which to create the resource.
+  If omitted, the provider-level region will be used. Changing this parameter will create a new resource.
+
+* `name` - (Required, String, ForceNew) Specifies the event name. The valid length is limited from `1` to `64`.
+  Only letters, digits, underscores (_) and hyphens (-) are allowed.
+
+  Changing this parameter will create a new resource.
+
+* `event_types` - (Required, List) Specifies the event type list. Valid values are:
+  + **SECRET_VERSION_CREATED**: Triggered when a version of a secret is created.
+  + **SECRET_VERSION_EXPIRED**: Triggered when a secret version expires. (Triggered only once for each expiry).
```

Comment:
the style of the descriptions should be same

Reply:
OK